sickbeard/show_queue.py

```diff
@@ -282,12 +283,35 @@ def run(self):
                 self._finishEarly()
                 return
         except Exception, e:
-            logger.log(u"Show name with ID %s don't exist in %s anymore. Please change/delete your local .nfo file or remove it from your TRAKT watchlist" %
+            logger.log(u"Show name with ID %s don't exist in %s anymore. it will be removed form your TRAKT watchlist" %
                 (self.indexer_id,sickbeard.indexerApi(self.indexer).name) , logger.ERROR)
```

Comment:
`s/form/from/`
also, please change it from `don't exist in` to `don't exist on` for proper english xD.